be/src/formats/parquet/group_reader.h

```diff
@@ -35,10 +35,12 @@ namespace starrocks::parquet {
 struct GroupReaderParam {
     struct Column {
         // column index in parquet file
-        int col_idx_in_parquet;
+        int32_t col_idx_in_parquet;
+
+        int32_t column_chunk_idx_in_parquet;
```

Comment:
what's the difference with `col_idx_in_parquet` this field?

Reply:
for example parquet meta:
col a,
col b: {
   col: b1,
   col: b2
},
col c

then col c's col_idx_in_parquet = 2，column_chunk_idx_in_parquet = 3